schema/schema.cc

```diff
@@ -1396,8 +1396,7 @@ schema_ptr schema_builder::build(schema::raw_schema& new_raw) {
     }
 
     if (static_props.is_group0_table) {
-        static constexpr auto schema_gc_grace = std::chrono::duration_cast<std::chrono::seconds>(std::chrono::days(7)).count();
-        new_raw._gc_grace_seconds = schema_gc_grace;
+        new_raw._gc_grace_seconds = 0;
```

Comment:
Yes ... the GC grace seconds shouldn't be used, but it shows in the "DESC SCHEMA" for the table (with the default value of 86400 when the code is removed completely). Thus this is just to show the value of 0 ... but if we are fine with showing the default value of 1 week there then this can be removed completely

Reply:
Removed the block completely (will keep the default GC grace seconds, but they will not apply anyway)